index.html

```diff
@@ -516,996 +448,45 @@ <h2>Profiling Mechanism</h2>
-              <tr>
-                <td>format</td>
-                <td>string</td>
-                <td>
-                  <span class="rfc2119-assertion" id="profile-5.1.4.2-thing-property-affordance-additional-constraints-6">
-                  If the field <code>format</code>
-                  is used, only formats defined in
-                  section 7.3.1-7.3.6 of
-                  [[JSON-SCHEMA]] MAY be used.
-                  </span>
-                </td>
-              </tr>
-              <tr>
-                <td>oneOf</td>
-                <td>string</td>
-                <td>
-                  <span class="rfc2119-assertion" id="profile-5.1.4.2-thing-property-affordance-additional-constraints-7">
-                  The DataSchema field <code>oneOf</code>
-                  does not make sense for properties
-                  and MUST NOT be used.
-                  </span>
-                </td>
-              </tr>
-              <tr>
-                <td>uriVariables</td>
-                <td>Map of DataSchema</td>
-                <td>
-                  <span class="rfc2119-assertion" id="profile-5.1.4.2-thing-property-affordance-additional-constraints-8">
-                  <code>uriVariables</code> MUST
-                  NOT be used.
-                  </span>
-                </td>
-              </tr>
-
-            </tbody>
-          </table>
-        </section>
-
-        <section>
-          <h4>Recommended Practice</h4>
-          <p>
-            It is highly RECOMMENDED to always specify a
-            <code>unit</code>
-            , if a value has a metric.
-            <span class="rfc2119-assertion" id="profile-5.1.4.3-thing-properties-recommended-practice-1">
-            Authors of <em>Thing Descriptions</em> should be aware, that units
-            that are common in their geographic region are
-            not globally applicable and may lead to
-            misinterpretation with drastic consequences.
-            </span>
-          </p>
-          <p>
-            The field
-            <code>unit</code>
-            could be used for non-decimal numeric types as
-            well, e.g. a string value with binary or hex
-            data (
-            <code>0xCAFEBABE</code>
-            ,
-            <code>0b01000010</code>
-            ), where the unit is
-            <code>hex</code>
-            or
-            <code>bin</code>
-            ,
-            <span class="rfc2119-assertion" id="profile-5.1.4.3-thing-properties-recommended-practice-2">
-            to indicate how the value should be
-            interpreted. It is strongly RECOMMENDED to use
-            the values
-            </span>
-            <code>hex</code>
-            ,
-            <code>oct</code>
-            or
-            <code>bin</code>
-            in this case to achieve interoperability.
-          </p>
-        </section>
-      </section>
-
+    This section defines the <a>Core Profile</a> by describing:
```

Comment:
The new language should be added, however removing the core data model is not appropriate, since it will break profile use cases.

Reply:
We discussed the use cases and motivation for the common data model in the October vF2F. Please see presentation slides: https://github.com/w3c/wot/blob/main/PRESENTATIONS/2021-10-online-f2f/2021-10-27-WoT-F2F-Architecture%2BProfiles-Lagally.pdf

Reply:
@mlagally I apologise for not being able to attend that meeting due to illness, but I see nothing in that presentation which convinces me that the data model constraints proposed in the current WoT Core Data Model section are necessary, or helpful. Please see my extensive comments in https://github.com/w3c/wot-profile/pull/93#pullrequestreview-717982575 from July in which I reiterate why I think they are undesirable and in some cases conflict with the agreed goal of the specification. Ege has said he agrees with those comments, and I note that you haven't yet responded to them.

I get the impression that you are the only member of the group who thinks that including these data model constraints in the Core Profile is a good idea, and that the majority of members of the group actually think that the majority of the proposed constraints aren't necessary. I am happy to continue to discuss this further to try to reach a consensus, but if we can't reach a consensus then according to the [Working Group's decision policy](https://www.w3.org/2020/01/wot-wg-charter.html#decisions), I understand the Chairs may call for a group vote.